### ISSUE TITLE
Block panel in combination with EXO/EKT app tag

### ISSUE THREAD

**Issue by annaengstrom** [Clinical-Genomics/cg#603]
customers are ordering the combination of panels (Capture kit) with EXO/EXT app tag

Problem: 
a panel should never be ordered with EXO/EXT app tag - check out https://clinical.scilifelab.se/requirements

Steps to reproduce
  1. Order EXO
  2. Choose GMCKsolid
  3. submit order
  4. order works

Expected outcome / suggested solution:
Block combination of Panel (UDF Capture Kit) and EXO/EXT app tag

Changing this manually for all samples requires a lot of work, and the option for capture kit + EXO/EXT app tag should never exist.

**Comment by annaengstrom** [comment]
FYI @annagellerbring

**Comment by annaengstrom** [comment]
@annagellerbring could you please inform cust002 not to order GMCKsolid for these samples?

**Comment by annagellerbring** [comment]
@annaengstrom yes it has been on my to-do list for a while. My plan was to do it the next time they order it, but I'll do it already now so we save some unnecessary work!

**Comment by annagellerbring** [comment]
> @annaengstrom yes it has been on my to-do list since last order when I discovered it. My plan was to do it the next time they order it, but I'll do it already now so we save some unnecessary work!

Communicated to cust002 in ticket #380665 2020-03-31.

**Comment by Mropat** [comment]
@henrikstranneheim is this still a problem?

**Comment by henrikstranneheim** [comment]
I do not know - what do you say @annaengstrom @annagellerbring ?

**Comment by henrikstranneheim** [comment]
Is this something that will be fixed when the mip panel pipeline is part of cg?

**Comment by annaengstrom** [comment]
This is an orderportal issue
